archivebox/parsers/pocket_api.py

```diff
@@ -0,0 +1,119 @@
+__package__ = 'archivebox.parsers'
+
+
+import re
+
+from typing import IO, Iterable, Optional
+from datetime import datetime
+from configparser import ConfigParser
+
+from pathlib import Path
+from pocket import Pocket
+import requests
+
+from ..index.schema import Link
+from ..util import (
+    enforce_types,
+)
+
+_COUNT_PER_PAGE = 500
+_SINCE_FILENAME = 'since.db'
```

Comment:
can you move the config import to the top of the file and set the full path here

```suggestion
_SINCE_FILENAME = Path(SOURCES_DIR) / 'pocket_api.db'
```